src/scenes/ProgressReports/Detail/ProgressReportDrawer/Steps/PromptVariant/VariantResponsesAccordion.tsx

```diff
@@ -0,0 +1,89 @@
+import { ExpandMore } from '@mui/icons-material';
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Typography,
+} from '@mui/material';
+import { SubmissionErrors } from 'final-form';
+import { DateTime } from 'luxon';
+import { useState } from 'react';
+import { Form } from 'react-final-form';
+import { SubmitButton } from '~/components/form';
+import { RichTextField, RichTextView } from '~/components/RichText';
+import { ProgressReportItemResponseEditFragment } from '../../ProgressReportDrawer.graphql';
+import { RoleIcon } from '../../RoleIcon';
+
+export const VariantResponsesAccordion = ({
+  response,
+  expanded: _expanded,
+  onSubmit,
+}: {
+  response: ProgressReportItemResponseEditFragment;
+  expanded?: boolean;
+  onSubmit: (input: any) => void | SubmissionErrors | Promise<SubmissionErrors>;
+}) => {
+  const [expanded, setExpanded] = useState(_expanded ?? false);
+  const [savedAt, setSavedAt] = useState<DateTime | null>(null);
+
+  if (response.response.canRead) {
+    return (
+      <Accordion
+        key={response.variant.responsibleRole}
+        expanded={expanded}
+        elevation={2}
+        square
+      >
+        <AccordionSummary
+          aria-controls={`${response.variant.responsibleRole}-content`}
+          expandIcon={<ExpandMore />}
+          sx={{
+            '& .MuiAccordionSummary-content': {
+              alignItems: 'center',
+            },
+          }}
+          onClick={() => setExpanded(!expanded)}
+        >
+          <RoleIcon roleStep={response.variant.responsibleRole} />
+          <span>{response.variant.label}</span>
+        </AccordionSummary>
+        <AccordionDetails sx={{ px: 4 }}>
+          {response.response.canEdit ? (
+            <Form
+              onSubmit={onSubmit}
+              initialValues={{
+                variant: response.variant.key,
+                response: response.response.value,
+              }}
+            >
+              {({ handleSubmit }) => (
+                <form onSubmit={handleSubmit}>
+                  <RichTextField name="response" label="Response" />
+                  {savedAt && (
+                    <Typography
+                      variant="caption"
+                      sx={{ mb: 1 }}
+                      component="div"
+                    >
+                      Saved at {savedAt.toISO()}
```

Comment:
```suggestion
                      Saved at <FormattedDateTime date={savedAt} />
```
I'll add this property to the API as well.

Reply:
The reason I was using ISO (and I agree that we can move away from it) was because it was providing seconds and milliseconds as well. Using the `FormattedDateTime` component does not provide that level of detail: `Saved at 11/8/2022, 9:20 AM`